.github/workflows/validate_data_sources.yml

```diff
@@ -47,15 +47,17 @@ jobs:
           NEW_RELIC_APP_NAME: ${{ secrets.NEW_RELIC_APP_NAME }}
           NEW_RELIC_HOST: staging-collector.newrelic.com
           NODE_ENV: production
+          PR_NUMBER: ${{ steps.get_pr_number.outputs.pr-number }}
         run: |
-          URL="https://api.github.com/repos/${GITHUB_REPOSITORY}/pulls/${{ steps.get_pr_number.outputs.pr-number }}/files"
+          URL="https://api.github.com/repos/${GITHUB_REPOSITORY}/pulls/$PR_NUMBER/files"
           DRY_RUN=true
           cd utils && yarn create-validate-data-sources $URL $DRY_RUN
 
       - name: Add commit status
         if: always()
+        env:
+          COMMIT_SHA: ${{ github.event.workflow_run.head_commit.id }}
         run: |
-          export COMMIT_SHA=$(echo '${{ toJSON(github.event) }}' | jq '.workflow_run.head_commit.id' | tr -d '"')
           curl \
             -X POST \
             --header 'authorization: Bearer ${{ secrets.GITHUB_TOKEN }}' \
```

Comment:
Does `{{ secrets.GITHUB_TOKEN }}` need to change to use the env variable instead?

Reply:
That value is generated by Github when the job runs, so it is considered trusted input. So we are not worried about it being escaped.